### ISSUE TITLE
aggregation methods failed with string labels of kind '01', '02', ... (converted to int 1, 2, 3)

### ISSUE THREAD

**Issue by alixdamman** [liam2/larray#522]
```python
    >>> arr = ndrange(Axis(['0{}'.format(i) for i in range(1, 5)], 'a'))
    >>> arr
    a  01  02  03  04
        0   1   2   3
    >>> arr.sum('01,02,03 >> 01_03; 04')
    ...
    ValueError: [1, 2, 3] is not a valid label for any axis
```

**Comment by gdementen** [comment]
This has little to do with aggregation methods. It is a problem with `__getitem__`, I think this is a variant of #200 . At least it is related to it.

**Comment by gdementen** [comment]
relates to and might be fixed by #595
